aws/table_aws_ssm_maintenance_window.go

```diff
@@ -0,0 +1,343 @@
+
+	return []string{aka}, nil
+}
+
+func getAwsSSMMaintenanceWindowTags(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getAwsSSMMaintenanceWindowTags")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	maintenanceWindowData := h.Item.(*ssm.GetMaintenanceWindowOutput)
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.ListTagsForResourceInput{
+		ResourceType: types.String("MaintenanceWindow"),
+		ResourceId:   maintenanceWindowData.WindowId,
+	}
+
+	// Get call
+	op, err := svc.ListTagsForResource(params)
+	if err != nil {
+		logger.Debug("getAwsSSMMaintenanceWindowTags", "ERROR", err)
+		return nil, err
+	}
+
+	return op, nil
+}
+
+func getMaintenanceWindowTargets(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getMaintenanceWindowTargets")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	maintenanceWindowData := h.Item.(*ssm.GetMaintenanceWindowOutput)
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.DescribeMaintenanceWindowTargetsInput{
+		WindowId: maintenanceWindowData.WindowId,
+	}
+
+	// Get call
+	op, err := svc.DescribeMaintenanceWindowTargets(params)
+	if err != nil {
+		logger.Debug("getMaintenanceWindowTargets", "ERROR", err)
+		return nil, err
+	}
+
+	return op, nil
+}
+
+func getMaintenanceWindowTasks(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getMaintenanceWindowTasks")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	maintenanceWindowData := h.Item.(*ssm.GetMaintenanceWindowOutput)
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.DescribeMaintenanceWindowTasksInput{
+		WindowId: maintenanceWindowData.WindowId,
+	}
+
+	// Get call
+	op, err := svc.DescribeMaintenanceWindowTasks(params)
+	if err != nil {
+		logger.Debug("getMaintenanceWindowTasks", "ERROR", err)
+		return nil, err
+	}
+
+	return op, nil
+}
+
+func ssmMaintenanceWindowTagListToTurbotTags(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("ssmMaintenanceWindowTagListToTurbotTags")
+	tagList := d.Value.([]*ssm.Tag)
+
```

Comment:
Return nil if no tag is set
```suggestion
  if tagList == nil {
    return nil, nil
  }
```